frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss

```diff
@@ -255,12 +264,16 @@
         height: 1.25rem;
         color: var(--muted);
 
-        &.LemonButtonWithSideAction__spacer--divider {
+        &--divider {
             padding: 0;
             margin-left: 0.25rem;
         }
     }
 
+    .LemonButton--small .LemonButtonWithSideAction__spacer--divider {
+        height: 1.125rem;
```

Comment:
Maybe beyond the scope of this PR but feels like this value could drift from https://github.com/PostHog/posthog/pull/19004/files#diff-c694fbbe6b6ff7273c964743f37751e5416009c150d0a9ddad1b4c031ef006d3R264. Perhaps the height should be a `calc` based on the button height minus some vertical padding